lib/vagrant-service-manager/binary_handlers/binary_handler.rb

```diff
@@ -100,7 +100,7 @@ def validate_url
         req.use_ssl = true if url.scheme == 'https'
         res = req.request_head(url.path)
 
-        unless res.code == '200'
+        unless %w(200 302).include? res.code
```

Comment:
So what is going on there? Is there ourdays a redirection in the oc download URL? I am pretty sure this was not the case before, right?